src/pull-request/pull-request-page.html

```diff
@@ -149,12 +149,17 @@
           reviewers: [
             {
               name: 'Thomas Smith',
-              lgtm: 'true',
+              lgtm: true,
               profile: 'https://avatars0.githubusercontent.com/u/5946422?v=3&s=72'
             },
             {
               name: 'Eva Anker',
-              lgtm: 'false' ,
+              lgtm: false ,
+              profile: 'https://avatars1.githubusercontent.com/u/5946438?v=3&s=72'
+            },
+            {
+              name: 'Eva Anker',
```

Comment:
Is this something we care about that sometimes there is a space and sometimes not before a comma.